examples/experimental/mnist_ptl/docker/Makefile

```diff
@@ -0,0 +1,2 @@
+build:
+	docker build -f Dockerfile -t ptl-image .
```

Comment:
nonblocking: we should add PTL to the environment image. Should do that in [a separate ticket](https://determinedai.atlassian.net/browse/DET-4986).